rpc/src/v1/helpers/errors.rs

```diff
@@ -57,6 +57,7 @@ mod codes {
 	pub const FETCH_ERROR: i64 = -32060;
 	pub const NO_LIGHT_PEERS: i64 = -32065;
 	pub const NO_PEERS: i64 = -32066;
+	pub const NO_LIGHT_TRANSACTIONS: i64 = -32066;
```

Comment:
I don't think this error code should be the same as the `NO_PEERS` one

Reply:
Thanks, yeah that's copy-paste error forgot the change the actual error code